src/pages/index.js

```diff
@@ -257,361 +291,348 @@ const Page = () => {
                 Transparent Data Encryption
-              </IndexCardLink>
-              <IndexCardLink to="/pg_extensions/ldap_sync">
+              </BannerWideLink>
+              <BannerWideLink to="/pg_extensions/ldap_sync">
                 EDB LDAP Sync
-              </IndexCardLink>
+              </BannerWideLink>
+
+              <BannerIconDivider
+                iconName={iconNames.INSTANCES}
+                headingText="Automation"
+              />
+
+              <BannerWideLink to="/tpa/latest/">
+                Trusted Postgres Architect
+              </BannerWideLink>
 
-              <span className="fw-bold text-muted text-uppercase small mt-4 d-block">
-                Extensions and Tools
-              </span>
+              <BannerIconDivider
+                iconName={iconNames.HIGH_AVAILABILITY}
+                headingText="High Availability"
+              />
 
-              <IndexCardLink to="/pg_extensions/">
+              <BannerWideLink to="/repmgr/latest">
+                Replication Manager (repmgr)
+              </BannerWideLink>
+              <BannerWideLink to="/supported-open-source/patroni/">
+                Patroni
+              </BannerWideLink>
+              <BannerWideLink to="/slony/latest">
+                Slony (Deprecated)
+              </BannerWideLink>
+              <BannerWideLink to="/supported-open-source/pglogical2/">
+                pglogical 2
+              </BannerWideLink>
+
+              <BannerWideLink to="/efm/latest">Failover Manager</BannerWideLink>
+
+              <BannerIconDivider
+                iconName={iconNames.BACKUP}
+                headingText="Backup and Recovery"
+              />
+
+              <BannerWideLink to="/supported-open-source/barman/">
+                Barman
+              </BannerWideLink>
+              <BannerWideLink to="/supported-open-source/pgbackrest/">
+                pgBackRest
+              </BannerWideLink>
+
+              <BannerIconDivider
+                iconName={iconNames.MIGRATION}
+                headingText="Migration"
+              />
+
+              {/* <BannerWideLink to="/migrating/oracle">
+                Migration Handbook
+              </BannerWideLink> */}
+              <BannerWideLink to="/migration_toolkit/latest">
+                Migration Toolkit
+              </BannerWideLink>
+              {/* <BannerWideLink to="/eprs/latest">
+                Replication Server
+              </BannerWideLink> */}
+            </BannerWideSubCard>
+
+            <BannerWideSubCard
+              iconName={iconNames.EXTENSION}
+              headingText="Extensions and Tools"
+            >
+              <BannerDivider headingText="Extensions" />
+
+              <BannerWideLink to="/pg_extensions/">
                 Supported Postgres extensions
-              </IndexCardLink>
+              </BannerWideLink>
+
+              <BannerWideLink to="//postgis/latest/">PostGIS</BannerWideLink>
```

Comment:
This link is not working on my local env, maybe drop one / ? 
```suggestion
              <BannerWideLink to="/postgis/latest/">PostGIS</BannerWideLink>
```